src/main/java/com/zeiterfassung/hwr/desktop/component/LoginPane.java

```diff
@@ -40,6 +42,11 @@
     private Button btnSubmit;
     private Label errorLabel;
 
+    public LoginPane(@Value("${spring.application.api.login}") String loginApi)
```

Comment:
Warum wird der Lokal in der Klasse gespeichert? Ist doch eine feste Globale Variable.

Reply:
Datenkapslung, wenn eine andere Klasse auch zugriff auf den Spring haben will soll sie sich den von application.properties holen und nicht von der LoginPane